stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1172,6 +1172,65 @@ public func _TFCGetCTensorHandleFromSwift(
   return handle.cTensorHandle
 }
 
+/// Adds `handle` as an input to `op`.
+@usableFromInline
+@_silgen_name("_swift_tfc_OpAddInputFromTensorHandle")
+func _TFCOpAddInputFromTensorHandle(_ op: CTFEOp,
+                                    _ handle: _AnyTensorHandle,
+                                    _ status: CTFStatus) {
+  TFE_OpAddInput(op, handle.cTensorHandle, status)
+}
+
+/// Adds `t` as an input or inputs to `op`.
+@usableFromInline
+@_silgen_name("_swift_tfc_OpAddInputFromTensorGroup")
+func _TFCOpAddInputFromTensorGroup<T: TensorGroup>(
+    _ op: CTFEOp, _ t: T, _ status: CTFStatus
+) -> Int32 {
+  let count = T._tensorHandleCount
+  let buffer = UnsafeMutablePointer<CTensorHandle>.allocate(capacity: Int(count))
```

Comment:
Instead of using an `UnsafeMutablePointer` and manually maintaining a `count`, I would suggest using `UnsafeMutableBufferPointer.allocate(capacity:)`, and you can then do a `for`-loop over it without having touching `count` again.